### ISSUE TITLE
Jon's Project Proposal: how did Mary Shelley's worldview change between the 1818 and 1831 editions of "Frankenstein"?

### ISSUE THREAD

**Issue by engeljonathan98** [obdurodon/dh_course#225]
In brief: we will compare the two most prominent versions of _Frankenstein_ to each other in order to quantify theories about Mary Shelley's changing points-of-view about agency and human existence.

Mary Shelley's seminal horror novel _Frankenstein: or, the Modern Prometheus_ is one of the best books in the English language and a personal favorite of mine. It tells of the scientist Victor Frankenstein and his ruination at the hands of his "creature", an experiment in creating life who feels rejected by his creator and sets out to destroy him.

But _why_ does the creature feel rejected? Whose _fault_ is that?

Shelley published multiple versions of the text throughout her life, the most well-known being the original publication in 1818 and the first mass-printed publication in 1831. The 1818 and 1831 texts are dramatically different from each other, with Shelley revising the book to reflect her own changing point-of-view.

[In her take on the two editions,](http://knarf.english.upenn.edu/Articles/mellor9.html) UCLA literary historian and theorist Anne K. Mellor argues that one main difference between the two texts is that Shelley reconsidered what is _behind_ Victor's fall. As Mellor sees it, in the 1818 text, Victor brings about his own end; he is an arrogant fool who is destroyed by his unwillingness to love his creation. In the 1831 edition, however, Mellor argues Shelley has experienced a fundamental shift in her worldview. Mellor believes Shelley came to see the world as a place of dominating natural forces and passions, in which human beings have no agency and are at the mercy of the unknowable schemes that motivate them. In other words, no free will.

The 1831 text, Mellor believes, reflect this. In this edition, Victor has no _real_ agency. Instead, he is an innocent victim of forces beyond his control. He is guided by chance and his naive curiosity down a path he could not possibly understand. Here, Mellor thinks, Victor is doomed not by _his own flaws_ but an unavoidable _destiny_.

This has massive implications on how Shelley saw human agency and existence, across all fields of philosophy and the human experience. So let's investigate that! If we take Mellor's theory as true, we might be able to _quantifiably_ track this change in tone by comparing the texts of both editions where they differ. The 1831 text would contain more content (which we can mark-up) that discusses destiny than the 1818 text and would reframe how Victor is discussed on a textual level.

**Comment by ebeshero** [comment]
@engeljonathan98 @djbpitt Yes--please be in touch! I notice that you've referenced the PA Electronic Edition (PAEE) of the 1818 and 1831 editions--that's an old wonderfully original project from the 1990s and served as the basis for our ongoing Variorum edition of Frankenstein. Over the past couple of years, one thing we did was correct some errors in the old PAEE (including a small oddly missing patch in the 1831 edition). We also added XML for a couple of distinct versions of the novel in between the 1818 and the 1831, and we're working with XML for the 1816 draft notebooks, encoded (quite differently) in page-by-page files by the Shelley-Godwin Archive. Anyway, we'd be really excited to see a team work with any of our XML files for your own project purposes--and happy to give you a running start on this.

**Comment by ebeshero** [comment]
There's kind of a lot going on in that repo that @djbpitt linked, but good XML base texts for the complete editions of 1818, "Thomas" copy, 1823, and 1831 editions are in this file-path in the repo: 

https://github.com/PghFrankenstein/Pittsburgh_Frankenstein/tree/master/collateXPrep/print-full
